lib/Mojo/WebSocketProxy/Backend/JobAsync.pm

```diff
@@ -13,6 +13,7 @@ use Job::Async;
 use JSON::MaybeUTF8 qw(encode_json_utf8 decode_json_utf8);
 use Log::Any qw($log);
 use MojoX::JSON::RPC::Client;
+use Syntax::Keyword::Try;
```

Comment:
if we're adding new modules, they'd need to be listed in `cpanfile` as well

Reply:
Added.